src/Storages/ObjectStorageQueue/ObjectStorageQueueMetadata.h

```diff
@@ -148,9 +150,17 @@ class ObjectStorageQueueMetadata
     /// Set local ref count for metadata.
     void setMetadataRefCount(std::atomic<size_t> & ref_count_) { chassert(!metadata_ref_count); metadata_ref_count = &ref_count_; }
 
+    void updateSettings(const SettingsChanges & changes);
+
+    std::pair<size_t, size_t> getCleanupInterval() const { return {cleanup_interval_min_ms, cleanup_interval_max_ms }; }
```

Comment:
nit: Since it returns a pair of `size_t`, it makes sense to specify units in the name: `getCleanupIntervalMs` or `getCleanupIntervalMilliseconds`.